app/helpers/application_helper.rb

```diff
@@ -39,7 +39,7 @@ def footer_links
   end
 
   def include_google_tag_manager?
-    cookies["consented-to-cookies"] == "yes" && Rails.configuration.app_role.production?
+    cookies["consented-to-analytics-cookies"] == "yes" && Rails.configuration.app_role.production?
```

Comment:
This area is under discussion:
I expect we will need to make changes on Google Tag manager side (and probably on this PR) regarding what to include/exclude depending on user consenting/not consenting to analytics/marketing cookies.